src/debug/di/shimremotedatatarget.cpp

```diff
@@ -215,8 +215,10 @@ ShimRemoteDataTarget::GetPlatform(
          *pPlatform = CORDB_PLATFORM_POSIX_X86;
      #elif defined(DBG_TARGET_AMD64)
          *pPlatform = CORDB_PLATFORM_POSIX_AMD64;
-    #elif defined(DBG_TARGET_ARM)
+     #elif defined(DBG_TARGET_ARM)
```

Comment:
I chose existing uniformity when I thought of it and not elsewhere.  I'm happy to drop whitespace, but while patching chose less patch noise here in hindsight.